src/components/BrowseResultsList/getBrowseResultsFormatter.js

```diff
@@ -138,8 +138,10 @@ const getBrowseResultsFormatter = ({
       return typeName || <NoValue />;
     },
     callNumber: r => {
-      if (r?.instance || r?.totalRecords) {
-        return getTargetRecord(r?.fullCallNumber, r, ...commonTargetRecordArgs);
+      const fullCallNumber = getFullCallNumber(r);
+
+      if (r?.totalRecords) {
+        return getTargetRecord(fullCallNumber, r, ...commonTargetRecordArgs);
```

Comment:
Should `<MissedMatchItem query={r.fullCallNumber} />` also be changed?

Reply:
No, in case of a missing item it will only have a `fullCallNumber` field which will be equal to the query